### PR TITLE
| Shaz | Restructure readme files

### PR DESCRIPTION
**Pull request by shaz-ahammed**

### CODE REVIEW

**Review by vraravam — CHANGES_REQUESTED** [review]
@shaz-ahammed  - please remove your customizations from this branch.

File: GettingStarted-Advanced.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
+The "Advanced" setup is the set of final steps to capture your application preferences (both system apps as well as custom apps) and back them up into an *encrypted remote repository*. Currently this kind of a service is offered by [keybase](https://keybase.io/) where you can get private, fully-encrypted repos for free.
```
Comment (vraravam) [inline GettingStarted-Advanced.md:1]: Not sure I agree that these lines need to move into the advanced section. "Advanced" is basically starting with the home folder as a git repo and the profiles folder as another git repo. Anything before that, I would consider, as part of the basic / common setup. wdyt?

File: GettingStarted-Basic.md [hunk 2]
```diff
@@ -32,20 +32,5 @@ All these scripts are optimized for fast loading of the shell so that the user c
    * Goto Preferences > Profiles > Default > Text (and change the font to 'MesloLGS Nerd Font')
    * Goto Preferences > Profiles > Default > Keys > Key Mappings > Presets (and choose 'Natural Text Editing')
 
-*These are optional based on your preferences:*
```
Comment (vraravam) [inline GettingStarted-Basic.md:35]: same as for prev file since these lines were effectively moved there.

File: GettingStarted-Basic.md [hunk 3]
```diff
@@ -32,20 +32,5 @@ All these scripts are optimized for fast loading of the shell so that the user c
...
-   * Search for 'Control Centre" and scroll down to 'Clock options' and change the built-in clock to show as analog to save horizontal space in the top menu bar
-   * Search for 'Full Disk Access' and add `iTerm', 'Terminal', 'zoom.us'
-   * Search for 'Camera' and add 'Arc', 'Brave', 'Firefox', 'zoom.us'
-   * Search for 'Microphone' and add 'Arc', 'Brave', 'Firefox', 'zoom.us'
-   * Search for 'Close and restore windows', and uncheck 'Close windows when quitting an application' (this will ensure that iTerm, Terminal, all browsers, etc - whichever have multiple windows open while quitting that application, will restore the same windows and tabs the next time you start that application.)
-   * Search for 'Default web browser' and set as per your preferences
-   * Search for 'iCloud' and login setup Desktop sync
-2. Open the `Finder` application and manually adjust the Finder sidebar preferences
-
+To continue with advanced setup [click here](GettingStarted-Advanced.md)
```
Comment (vraravam) [inline GettingStarted-Basic.md:35]: i like this!

File: README.md [hunk 4]
```diff
@@ -1,31 +1,24 @@
...
+- [Click here to get started with the basic setup](#click-here-to-get-started-with-the-basic-setup)
 
 # Background
 
-This repo was created from the gists that I was using to help others adopt my machine setup. It contains the common dotfiles as well as the scripts that I use to share the machine setup.
-
-These scripts are idempotent and can run on a vanilla OS as well as once the whole setup has been completed.
-
-Each script will warn the users if its skipping a step, and if you want to rerun the script but force that step, you just need to delete the control `if` condition (you should have a basic understanding of shell programming to figure out what to delete/how to force without bypass).
+This repo was created to backup and easily remage a new system. Should add a breif about the intend/purpose of this repo
```
Comment (vraravam) [inline README.md:8]: i like this!

Comment (vraravam) [reply README.md:8]: typo: `remage` --> `re-image`

File: README.md [hunk 5]
```diff
@@ -35,19 +28,7 @@ Most of the folder structures are governed by the following environment variable
 
 if you want to capture data from your current mac, please follow the instructions [here](Prerequisites.md)
 
-# Basic setup
```
Comment (vraravam) [inline README.md:38]: not sure why you want to delete these lines. imo, they set the ground rules / intention about the 2-stage backup - which in turn allows ppl to stop with step 1 if they choose to.

File: README.md [hunk 6]
```diff
@@ -64,6 +45,4 @@ The backup strategy is **not a one-off activity**. It will require you to take s
 * Ensure that the git repo catalogs that you are "tracking" in the `${PERSONAL_CONFIGS_DIR}/repositories-*.yml` files are kept up-to-date so that resurrection in your new machine will be seamless
 * Ensure to run the `scripts/capture-defaults.sh` (with the export switch) to export and capture/backup your preferences for all installed applications from your current machine
 
-# Extras/Details
-
-Some utility scripts have been provided in this repo - which you can use to manage the backup strategy in a better fashion. Details can be found [here](Extras.md)
+# [Click here](GettingStarted-Basic.md) to get started with the basic setup
```
Comment (vraravam) [inline README.md:48]: i see your point about taking them through a journey/story-telling style. If so, then the table of contents on the top of the README might also need to be removed?

**Review by vraravam — CHANGES_REQUESTED** [review]

**Comment by vraravam** [comment]
@shaz-ahammed  - could you please rebase from the latest `master` branch and see if there's any changes that you still think are pertinent for this PR?

File: GettingStarted-Advanced.md (hunk 1 above)
Comment (shaz-ahammed) [reply GettingStarted-Advanced.md:1]: intend was to keep the basic setup setup instructions as minimal as possible, but yeah i agree

Comment (vraravam) [reply GettingStarted-Advanced.md:1]: I agree that this can be the starting para in the Advanced file.

**Review by vraravam — CHANGES_REQUESTED** [review]

**Comment by vraravam** [comment]
ok - after reviewing once more, it doesn't look like any major changes are being made in this PR. Closing

**Pull request closed without merging**